standards/B3.md

```diff
@@ -0,0 +1,228 @@
+# ERC948 Interface for Pull Requests - B3's Financial Transaction Business Process Protocol (FTBPP)
+### Summary
+A standard interface to be implemented by smart wallets that wish to allow pre-authorized pull requests.
+
+### Abstract
+This standard defines an interface to pre-authorize payments in order to allow parties to fulfill the payment at a later date via a pull request. The approval/pull request mechanism is purposefully generalized so that it can be used for both recurring pull requests (subscriptions) and one-off pull requests. This standard allows for payment to occur in Ether, ERC20, ERC223 tokens. This should be integrated into future versions of smart wallets, allowing for a more seamless process.
+
+### Our Thoughts
+The ability to support all types of pull requests, not only subscriptions, is of utmost importance. Subscription is just one subset of an abundance of use cases for pull requests. Therefore, because we cannot now predict all of the unique uses for pull requests, it is important to create a standard that is general, yet secure enough to allow creative developers to build upon without the need for creating a new standard. 
+
+### Motivation
+It is inconvenient for a user to make a push payment every time a payment is required. The legacy banking system supports automatic bill payment features, allowing users to preauthorize recurring payments, saving users both the time to make the payment and the hassle of remembering when payments are due. Until we reduce the friction of subscriptions on the ethereum blockchain, companies will have to choose between customer experience and a subscription monetization model.
+
+Finally, it is important that there is a standard interface for pull requests. Having a standard allows wallets to create a pleasant UI/UX for its users, allowing users to feel comfortable and safe while authorizing, rejecting, and managing their pull requests.
+
+## Specification
+
+### Definitions
+* planID - a unique hash used to identify a pull request plan
+* planAgent - the authorized address to initiate a specific pull request (planID)
+
+### Features
+* The Smart Wallet Interface only handles the approval, revocation of approval, and transfer of the asset. Therefore, the interface allows the approved planAgent to handle the complexity of determining the quantity, type of asset, recurrence, etc.. Thus, by managing the core functionality shared by all pull requests, the interface is interoperable with all varieties of complex pull requests.
+* Supports pull requests of Ether, ERC20, ERC223 tokens
+* Directly transfers funds from owner of the smart wallet to recipient
+
+## Methods
+
+### approvePullRequest
+
+Called by the owner of the smart wallet.
+
+* @param planAgent - it is the address of a smart contract (possibly a core account too)
+* @param recurrency - limits the number of times a planAgent can call ‘pull’ in a given period
```

Comment:
This is somewhat limiting, at least from the user perspective, the provider can pull every 5min if he wants (the due amount for the period). 
Given the `allowance` pattern (user allows up to some amount, but can also spend the allowance) might be beneficial to capture the payment as often as needed.